.github/workflows/tox-test.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     strategy:
```

Comment:
Maybe do the same in `tox.ini`?

Reply:
I'm not so sure about that, I don't think we shouldn't even have support the intermediate versions in tox, we just don't need to be running those in CI, when you test locally, you should have the option though.